client/src/components/Visualization/charts/ScatterPlot.js

```diff
@@ -49,14 +49,19 @@ function formatAdvertiserCountSnapshot(snap, year) {
  * Returns the exclusive bounds for the range of all data points being charted.
  * Uses the spread `...` operator.
  * `https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax` 
+ * If the list is empty a range [0,1] will be returned. 
```

Comment:
Is the empty list an excepted behaviour? Why are you returning a range instead of throwing an error?

Reply:
Expected behavior. See PR doc for background.